### DISCUSSION TITLE
How to create an index.html file on Git (vim)

### DISCUSSION THREAD

**Discussion by Kevin-Boateng** [community/community#73466] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi all, 

I am new to version control. I am currently attempting to complete an assignment. I've been asked to create an html file and I was provided source code, but the instructions are not very clear on how to do this. I managed to create the css/app.css folder and the js/app.js folder. Could someone please assist in this?

Thanks,
Kevin

**Comment by ThilanWijesinghe** [comment]
You can get this badge by merging 2 pull requests

**Comment by ThilanWijesinghe** [comment]
Create a New Directory for Your Project:
You should start by creating a new directory (folder) for your project. You can do this using your operating system's file explorer or by using a command in the terminal.

For example, if you're using the command line, you can create a directory like this:

bash
Copy code
mkdir your-project-name
cd your-project-name
Initialize a Git Repository:
To track changes in your project and utilize version control, you'll want to set up a Git repository. Run the following command in your project directory:

bash
Copy code
git init
Organize Your Folders:
You mentioned you already have css/app.css and js/app.js folders. Make sure they are located within your project directory.

Create an HTML File:
If you've been provided with the HTML source code, you can create an HTML file (e.g., index.html) in your project directory and paste the provided HTML code into it. You can use a text editor or code editor like Visual Studio Code, Sublime Text, or Notepad++ to create and edit this file.

Add Files to Version Control:
Use the following commands to add your HTML, CSS, and JavaScript files to the Git repository:

bash
Copy code
git add index.html
git add css/app.css
git add js/app.js
Commit Your Changes:
After adding the files, you need to make a commit to record the changes in the repository:

bash
Copy code
git commit -m "Initial commit"
Replace "Initial commit" with a brief description of what you did.

Set Up a Remote Repository (Optional):
If your assignment requires you to collaborate with others or if you want to store your project on a remote platform like GitHub, you can create a remote repository and connect it to your local repository. Instructions for this step may vary depending on the platform you're using.

Push Your Code (Optional):
If you're using a remote repository, you can push your code to the remote server. Here's an example using GitHub:

bash
Copy code
git remote add origin your-github-repo-url
git branch -M main
git push -u origin main

**Reply by Kevin-Boateng** [reply]
Thank you so much, I've been stuck on this for quite some time. I'm currently using Windows, does it matter if I use Note ++, vim, or VS code? It's currently set to use Note ++ but I would like to change it to vs code, but I am not sure how to go about doing this with Git.
